app/components/split-view.js

```diff
@@ -159,10 +159,19 @@ export default Ember.Component.extend({
   },
 
   minChildPercentage: function(view) {
+    var cssInt = function(name) {
+      return parseInt(view.$().css(name));
+    }
     if(this.get('isVertical')) {
-      return parseInt(view.$().css("min-width")) / this.get('width') * 100 + this.get('sash.widthPercentage') / 2;
+      return (cssInt("min-width") + cssInt("padding-left") + cssInt("padding-right")
```

Comment:
From w3schools:
Definition and Usage
The width property sets the width of an element.

Note: The width property does not include padding, borders, or margins; it sets the width of the area inside the padding, border, and margin of the element!